GPU/GPUTracking/SliceTracker/GPUTPCTrackletSelector.cxx

```diff
@@ -62,7 +64,7 @@ GPUdii() void GPUTPCTrackletSelector::Thread<0>(int nBlocks, int nThreads, int i
       if (ih != CALINK_INVAL && ih != CALINK_DEAD_CHANNEL) {
         GPUglobalref() const MEM_GLOBAL(GPUTPCRow)& row = tracker.Row(irow);
         bool own = (tracker.HitWeight(row, ih) <= w);
-        bool sharedOK = ((nShared < nHits * kMaxShared));
+        bool sharedOK = nShared < (nHits < sharingMinNorm ? maxShared : nHits * maxSharedFrac);
```

Comment:
as mentioned some time before, I would change this to `nShared <= ...`, to allow the first cluster to be shared.

Reply:
ok, originally I had also `sharingMinNorm` set to at least 1, which made it equivalent to `nShared <= ...` but it was somehow lost when suppressing cluster weights modifications. Anyway, like that it is probably clear.